libs/model/src/community/SetCommunityStake.command.ts

```diff
@@ -1,25 +1,19 @@
-import { CommandMetadata, InvalidState } from '@hicommonwealth/core';
-import { z } from 'zod';
+import {
+  CommandMetadata,
+  InvalidState,
+  setCommunityStakeSchema,
```

Comment:
import {schemas} from "@hicommonwealth/core"

use schemas.community.SetCommunityStake ... we are using the Pascal case convention for artifact names... and camel for method names... do you think we should change this?

Reply:
Yeah not sure about the naming conventions. Ideally I would like to have everything camelCase and only classes PascalCase since it matches front end, but it is not so important to me, as long as we stay consistent

Reply:
we can assume types and schemas are similar to classes... types are also Pascal in TS

Reply:
same with artifact factories and metadata